app/controllers/events/view/tickets/orders/list.js

```diff
@@ -35,6 +35,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         cellComponent   : 'ui-table/cell/events/view/tickets/orders/cell-date',
         headerComponent : 'tables/headers/sort',
         width           : 100,
+        dateFormat      : 'D MMM, YYYY h:mm A (z)',
```

Comment:
As we discussed in the meeting, there is no need of timezone if the date is not event/session date